packages/server/api/src/app/ee/projects/project-usage-service.ts

```diff
@@ -0,0 +1,100 @@
+import { AppSystemProp, exceptionHandler } from '@activepieces/server-shared'
+import { ApEdition, ApEnvironment, getCurrentBillingPeriodEnd, getCurrentBillingPeriodStart, isNil, ProjectUsage, UsageEntityType, UsageMetric } from '@activepieces/shared'
+import { FastifyBaseLogger } from 'fastify'
+import { getRedisConnection } from '../../database/redis-connection'
+import { system } from '../../helper/system/system'
+import { getUsage, redisKeyGenerator } from '../../helper/usage'
+import { platformService } from '../../platform/platform.service'
+import { projectService } from '../../project/project-service'
+import { userInvitationsService } from '../../user-invitations/user-invitation.service'
+import { platformBillingService } from '../platform-billing/platform-billing.service'
+import { platformUsageService } from '../platform-billing/platform-usage-service'
+import { projectMemberService } from '../project-members/project-member.service'
+import { projectLimitsService } from '../project-plan/project-plan.service'
+
+const environment = system.get(AppSystemProp.ENVIRONMENT)
+const edition = system.getEdition()
+
+export const projectUsageService = (log: FastifyBaseLogger) => ({
+    async getProjectUsageForBillingPeriod(projectId: string): Promise<ProjectUsage> {
+        const startBillingPeriod = getCurrentBillingPeriodStart()
+
+        const tasks = await getUsage(projectId, UsageEntityType.PROJECT, startBillingPeriod, UsageMetric.TASKS)
+        const aiCredit = await getUsage(projectId, UsageEntityType.PROJECT, startBillingPeriod, UsageMetric.AI_CREDIT)
+        const tables = await getUsage(projectId, UsageEntityType.PROJECT, startBillingPeriod, UsageMetric.TABLES)
+        const mcpServers = await getUsage(projectId, UsageEntityType.PROJECT, startBillingPeriod, UsageMetric.MCP_SERVERS)
+        const activeFlows = await getUsage(projectId, UsageEntityType.PROJECT, startBillingPeriod, UsageMetric.ACTIVE_FLOWS)
+
+        const projectMembers = await projectMemberService(log).countTeamMembers(projectId)
+        const projectInvitations = await userInvitationsService(log).countByProjectId(projectId)
```

Comment:
I don't think we use the teamMembers anymore tbh in the billing

Just remove them (and If we don't use countTeamMember remove it as well)